lib/hanami/controller/configuration.rb

```diff
@@ -416,7 +416,7 @@ def format(hash)
       def mime_types
         @mime_types ||= begin
                           ((@formats.keys - DEFAULT_FORMATS.keys) +
-                          ::Rack::Mime::MIME_TYPES.values).freeze
+                          Hanami::Action::Mime::MIME_TYPES.values).freeze
```

Comment:
My bad, I totally forgot about that. Any suggestions on what to move to the top of the list? My bet would be `html, txt, json, pdf, zip` in that order.